scripts/check_format.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+diff="$(opa fmt -d lib/ validator/)"
+
+if [[ "$diff" == "" ]]; then
```

Comment:
I would rather check the return code for the previous command ($? -ne 0)

Reply:
it exits 0 diff or no diff.

Reply:
I believe you can redirect the error to the standard output:

diff="$(echo opa fmt -d lib/ validator/ 2>&1)"

if [[ $? -ne 0 ]];
then

Reply:
If there is a diff, it exits 0.  If there is no diff, it exits 0.  We can't rely on the exit code for whether or not there is a diff.

Reply:
sounds good. it's too bad opa does not behave in a typical way